onnx2pytorch/operations/loop.py

```diff
@@ -0,0 +1,187 @@
+from collections import defaultdict
+from copy import deepcopy
+from functools import partial
+import warnings
+
+import numpy as np
+import onnx
+import torch
+from onnx import numpy_helper
+from torch import nn
+from torch.nn.modules.linear import Identity
+
+from onnx2pytorch.utils import (
+    get_inputs_names,
+    get_outputs_names,
+)
+
+
+class Loop(nn.Module):
+    def __init__(
+        self,
+        opset_version,
+        batch_dim,
+        body: onnx.GraphProto,
+    ):
+        from onnx2pytorch.convert.operations import (
```

Comment:
These delayed imports are to avoid Python import errors due to circular import from a partially-initialized module. `operations.convert_operations` needs to import `Loop`, but `Loop` also needs to import `operations.convert_operations` to convert the operations inside the subgraph for the body of the loop. Meanwhile, `ConvertModel.__init__` needs `operations.convert_operations`, while `ConvertModel.forward` needs `Loop`.

Reply:
What about doing here: 
```
self.ops = import_module("onnx2pytorch.convert.operations")
self.c = import_module("onnx2pytorch.constants")
```
And then use it in forward method.

In the beginning of the file you should add: `from importlib import import_module`